test/bigtest/helpers/wait.js

```diff
@@ -0,0 +1,3 @@
+const wait = (ms = 1000) => new Promise(resolve => { setTimeout(resolve, ms); });
+
+export default wait;
```

Comment:
@Godlevskyi, yes, I used it [here](https://github.com/folio-org/ui-eholdings/pull/817/files#diff-8630ad1a1d8f9ad1798b76413cf13a67R216), because a debounced function is used in [NotesAssigningModal](https://github.com/folio-org/stripes-smart-components/blob/master/lib/Notes/components/NotesAccordion/components/NotesAssigningModal/NotesAssigningModal.js#L232)